app/services/event.service.js

```diff
@@ -273,7 +280,11 @@
      */
     function splitNode(response,unix,filterString){
       var filteredList = response.data;
-
+      // Check Start Date and End Date. Only needed for All filter
+      var start = new Date(filterString.split('filter[date][value][0]=')[1].split('&')[0].split(' ')[0]).getTime() / 1000;
+      if( filterString.includes('filter[date][value][1]=') ){          
```

Comment:
.includes is not available in IE. Change to `filterString.indexOf('filter[date][value][1]=') > -1`

Reply:
dang it IE